### DISCUSSION TITLE
https error 400

### DISCUSSION THREAD

**Discussion by Gawatkids** [Budibase/budibase#13649] [Help]
When I try uploading a file to my repository the page crashes and shows 
This page isn’t working
If the problem continues, contact the site owner.
HTTP ERROR 400

**Comment by mikesealey** [comment]
Hi @Gawatkids 

Please talk me through your workflow - When you say you are trying to upload a file to your repository, are you making an open-source code contribution to the GitHub repository?

 Or are you using a form with a File Upload component within Budibase?
 
 Can you provide a screenshot, or better yet capture it with the [Jam chrome extension](https://jam.dev/?invite=8386b9c5-8f83-4598-99fe-8e3e7eb8bc9f) and share here?
